poly-synth/src/poly-synth.js

```diff
@@ -32,15 +32,38 @@ class PolySynth {
     this.releasedVoices_ = [];
 
     // The maximum and minimum cutoffs are experimentally determined.
-    this.lowPassMaxCutoff = 16000;
-    this.lowPassMinCutoff = 60;
+    this.maxCutoff = 16000;
```

Comment:
The synthesizers I've looked at don't distinguish if its using a low pass or a high pass filter unless it exposes both, so partly I didn't think it was necessary and partly, the code reads better with this shorter variable name.